src/QirRuntime/test/QIR-static/compiler/QirTarget.qs

```diff
@@ -1,135 +1,63 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-namespace Microsoft.Quantum.Instructions {
-
-    operation S (qb : Qubit) : Unit {
-        body intrinsic;
-    }
-
-    operation Rx (theta : Double, qb : Qubit) : Unit {
-        body intrinsic;
-    }
-
-    operation Rz (theta : Double, qb : Qubit) : Unit {
-        body intrinsic;
-    }
-}
-
 namespace Microsoft.Quantum.Intrinsic {
```

Comment:
This file is almost the same as the QSharp.Core target package, particularly the list of files defined in this props file: https://github.com/microsoft/qsharp-runtime/blob/main/src/Simulation/TargetDefinitions/TargetPackages/QSharpCore.Package.props#L6. I wonder if there is a way to pull that in here without bringing all the other dependencies such that you don't have to repeat them here. Perhaps in a later PR there will be a chance to remove this file in favor of using the target package?

The biggest difference is the presence of the `open Microsoft.Quantum.Targeting` but I'm not sure that is still needed?

Reply:
I've logged https://github.com/microsoft/qsharp-runtime/issues/498 about automating qir files generation. Part of that work will be to get rid of the custom core/target.qs files in the runtime.